website/src/components/CardLink/styles.module.css

```
@@ -46,6 +47,7 @@
 
 .cardDescription {
 	font-size: 0.8rem;
+	text-wrap: wrap;
```

```suggestion
	text-wrap: wrap;
	text-wrap: pretty;
```
On browsers which support 'pretty' then it will probably look slightly nicer